js/plugins/TimeFly.js

```diff
@@ -217,17 +58,11 @@
   const display = () => frames;
 
   const displayTime = () => {
-    const s = getSeconds().toString().padStart(2, 0);
-    const m = getMinutes().toString().padStart(2, 0);
-    const H = getHours().toString().padStart(2, 0);
-    return `${H}:${m}:${s}`;
+    return (new RM_Timestamp(frames)).displayTime();
```

Comment:
```suggestion
    return (new RM_Timestamp(frames)).displayTime();
```
对于几乎所有游戏来说, 小时分钟秒的结构是统一不会变的, 所以Timefly应该可以独立完成这部分功能的.

Reply:
确实，TimeFly并不涉及日期的概念